Kickstarter-iOS/AppDelegate.swift

```diff
@@ -211,7 +211,6 @@ internal final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     return self.viewModel.outputs.applicationDidFinishLaunchingReturnValue
   }
-  // swiftlint:enable function_body_length
```

Comment:
While this PR was goin on swiftlint 0.22.0 dropped, which now checks for stale linter rules. It doesn't find these trailing ones, so I did a quick manual check based on our disabled rules.

Reply:
Nice, thanks!